bot.py

```diff
@@ -464,6 +470,11 @@ async def myresults(context: commands.Context) -> None:
             score[1] += 1
     
     message = []
+    message.append("Please keep in mind that music is subjective, and that "
+                   "these scores shouldn't be taken to represent the quality of"
+                   " your entry-- your artistic work is valuable, regardless of"
+                   " what results it was awarded, so don't worry too much about it")
+    message.append("And with that out of the way")
```

Comment:
```suggestion
    message.append("And with that out of the way...")
```
Presentation matters ;)

Reply:
and apparently im very bad at it